components/Card.tsx

```diff
@@ -0,0 +1,14 @@
+import next from "next";
+import React from "react";
+
+
+
+export default function Card (props:any) {
+  return (
+    <div className="border border-none w-56 h-72 ">
+        <img src={props.img} className=" mx-auto mt-3" />
```

Comment:
- Use ```<Image />``` tag from ```next/image``` instead of using html ```<img />``` tag

Reply:
The problem was solved and the tags changed